### PR TITLE
[url-file-size] add types

### PR DESCRIPTION
**Pull request by shahoob**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/url-file-size/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Type definitions for url-file-size 1.0
+// Project: https://github.com/dimdenGD/url-file-size#readme
+// Definitions by: Shehab Ali Tweel <https://github.com/shahoob>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference lib="DOM" />
```
Comment (jakebailey) [inline types/url-file-size/index.d.ts:6]: This doesn't seem right; the readme of the package shows using Node's URL, not the browser URL.

**Review by jakebailey — CHANGES_REQUESTED** [review]

File: types/url-file-size/index.d.ts (hunk 1 above)
Comment (shahoob) [reply types/url-file-size/index.d.ts:6]: yes, i don't know how do i tell DefinitelyTyped that i need @types/node.

Comment (jakebailey) [reply types/url-file-size/index.d.ts:6]:
```ts
/// <reference types="node" />

import { Url } from 'url';
```

Comment (shahoob) [reply types/url-file-size/index.d.ts:6]: ok thanks

Comment (jakebailey) [reply types/url-file-size/index.d.ts]: You imported it, but you aren't actually using it; not sure how that compiles. maybe you don't need either?

Comment (shahoob) [reply types/url-file-size/index.d.ts]: the source code uses the URL class, not the interface.
i updated the definition files to reflect that.

**Review by jakebailey — APPROVED** [review]

**Comment by shahoob** [comment]
Ready to merge

**Pull request merged**